lib/pbench/agent/results.py

```diff
@@ -311,20 +311,23 @@ def __init__(
         self.upload_url = f"{server_rest_url}/upload/{tbname}"
         self.logger = logger
 
-    def copy_result_tb(self, token: str) -> None:
+    def copy_result_tb(self, token: str, access: str) -> None:
```

Comment:
Can you make this parameter optional?  Then only when `--access` is specified with a value include the parameters on the request?

Further, I think we should validate the access parameter if present as having a value of `public` or `private`.

Reply:
if 

> Can you make this parameter optional? Then only when `--access` is specified with a value include the parameters on the request?

if we are using click to use `private` as default value then `either` public or `private` will be passed as access values. I don't think there will be any scenario when this value will not include any parameters. Although if we want to move away from using click for `default` values then we can for sure go for that.